### ISSUE TITLE
[Alert] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#294]
## Story description

- [ ] Update component documentation page with corrected color specs

### React
```
overlayColor: globalTokens.black
infoColor: globalTokens.lightBlue
successColor: globalTokens.lightGreen
warningColor: globalTokens.lightYellow
errorColor: globalTokens.lightPink
focusColor: globalTokens.blue
titleFontColor: globalTokens.black
contentFontColor: globalTokens.black
boxShadowColor: #00000012
borderColor: "transparent"
```
   
### Angular (Pending Update)

```
  "--alert-infoColor": globalTokens.lightBlue,
  "--alert-confirmColor": globalTokens.lightGreen,
  "--alert-warningColor": globalTokens.lightYellow,
  "--alert-errorColor": globalTokens.lightPink,
  "--alert-overlayColor": globalTokens.black,
  "--alert-focusColor": globalTokens.blue, //ponerlo en el componente
```

## Alert color specs table

|   Component token     |   Element                     |   Core token                  |   Value       |
| --------------------- | ----------------------------- | ----------------------------- | ------------- |
|   `infoIconColor`         |   Icon       |   `Hal-Blue-S-35`    |   `#0067b3`   |
|   `successIconColor`      |   Icon        |   `Hal-Green-S-39`   |   `#24a148`   |
|   `warningIconColor`      |   Icon        |   `Hal-Yellow-D-40`  |   `#c59f07`   |
|   `errorIconColor`        |   Icon       |   `Hal-Red-S-41`    |   `#d0011b`   |
|   `infoBackgroundColor`         |   Container background        |   `Hal-Blue-L-95`    |   `#e6f4ff`   |
|   `successBackgroundColor`      |   Container background        |   `Hal-Green-L-95`   |   `#eafaef`   |
|   `warningBackgroundColor`      |   Container background        |   `Hal-Yellow-L-95`  |   `#fef9e6`   |
|   `errorBackgroundColor`        |   Container background        |   `Hal-Red-L-95`    |   `#ffe6e9`   |
|   `infoBorderColor`             |   Container border            |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `successBorderColor`     |   Container border            |   `Hal-Green-S-39`            |   `#24a148`   |
|   `warningBorderColor`      |   Container border            |   `Hal-Yellow-S-57`           |   `#f7cf2b`   |
|   `errorBorderColor`            |   Container border            |   `Hal-Red-S-41`              |   `#d0011b`   |
|   `titleFontColor`    |   Label                       |   `Hal-Black`                 |   `#000000`   |
|   `contentFontColor`  |   Content                     |   `Hal-Black`                 |   `#000000`   |
|   `focusColor`        |   Action:focus            |   `Hal-Blue-S-35`         |   `#0067b3`   |
|   `boxShadowColor`    |   Container                   |   -                           |   `#00000012` |
|   `overlayColor`      |   Overlay                  |   `-`                 |   `#000000B3`   |

**Comment by mnnaveenk** [comment]
## react

overlayColor: globalTokens.black
infoColor: globalTokens.lightBlue
successColor: globalTokens.lightGreen
warningColor: globalTokens.lightYellow
errorColor: globalTokens.lightPink
focusColor: globalTokens.blue
titleFontColor: globalTokens.black
contentFontColor: globalTokens.black
boxShadowColor: #00000012
borderColor: "transparent"
   

## Angular

  "--alert-infoColor": globalTokens.lightBlue,
  "--alert-confirmColor": globalTokens.lightGreen,
  "--alert-warningColor": globalTokens.lightYellow,
  "--alert-errorColor": globalTokens.lightPink,
  "--alert-overlayColor": globalTokens.black,
  "--alert-focusColor": globalTokens.blue, //ponerlo en el componente

## Alert color specs table


|   Component token     |   Element                     |   Core token                  |   Value       |
| --------------------- | ----------------------------- | ----------------------------- | ------------- |
|   `overlayColor`      |   Container                   |   `Hal-Black`                 |   `#000000`   |
|   `infoColor`         |   Container Background        |   `globalTokens.lightBlue`    |   `#CEE0F5`   |
|   `successColor`      |   Container Background        |   `globalTokens.lightGreen`   |   `#DBF1C4`   |
|   `warningColor`      |   Container Background        |   `globalTokens.lightYellow`  |   `#FCF2BD`   |
|   `errorColor`        |   Container Background        |   `globalTokens.lightPink`    |   `#F9CFCF`   |
|   `focusColor`        |   Container: Focus            |   `globalTokens.blue`         |   `#005FCC`   |
|   `titleFontColor`    |   Label                       |   `Hal-Black`                 |   `#000000`   |
|   `contentFontColor`  |   Content                     |   `Hal-Black`                 |   `#000000`   |
|   `boxShadowColor`    |   Container                   |   -                           |   `#00000012` |
|   `borderColor`       |   Container                   |   `transparent`               |   -           |
|   `confirmColor`      |   Container                   |   `globalTokens.lightGreen`   |   `#DBF1C4`   |

**Comment by aweell** [comment]
Added `borderColor` tokens:

|   Component token     |   Element                                   |   Core token                            |   Value       |
| -------------------------- | ------------------------ | ----------------------------- | ------------- |
|   `infoBorderColor`             |   Container border            |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `successBorderColor`     |   Container border            |   `Hal-Green-S-39`            |   `#24a148`   |
|   `warningBorderColor`      |   Container border            |   `Hal-Yellow-S-57`           |   `#f7cf2b`   |
|   `errorBorderColor`            |   Container border            |   `Hal-Red-S-41`              |   `#d0011b`   |

**Comment by aweell** [comment]
Closed via #374
